manifests/reports.pp

```diff
@@ -22,35 +22,43 @@
   $puppet_gem_provider = $datadog_agent::params::gem_provider,
 ) inherits datadog_agent::params {
 
-  include datadog_agent
-  $rubydev_package = $datadog_agent::params::rubydev_package
-
-  # check to make sure that you're not installing rubydev somewhere else
-  if ! defined(Package[$rubydev_package]) {
-    package {$rubydev_package:
-      ensure => installed,
-      before => Package['dogapi']
+  if ($::operatingsystem == 'Windows') {
+
+    fail('Reports are not supported on Windows')
```

Comment:
I think this message is a tad bit misleading - windows puppet clients will generate their reports, send them to the `master`, and the `master` node should report them. I guess what we're trying to say here something like _"Reporting is not supported from a windows host"_ (or something along those lines)?